tests/Event/Productions/ProductionEnrichedEventRepositoryTest.php

```diff
@@ -106,4 +104,45 @@ public function it_returns_production_data_when_event_belongs_to_a_production():
         $this->assertEquals($productionName, $fetchActual->getBody()->production->title);
         $this->assertEquals(['foo/' . $otherEventId], $fetchActual->getBody()->production->otherEvents);
     }
+
+    /**
+     * @test
+     */
+    public function it_does_not_save_a_production_in_json_document(): void
+    {
+        $inMemoryDocumentRepository = new InMemoryDocumentRepository();
+        $newProductionEnrichedEventRepository = new ProductionEnrichedEventRepository(
```

Comment:
Very small nit, I think this can just be called `productionEnrichedEventRepository`